### PR TITLE
Initial docs content for intro

### PR DESCRIPTION
**Pull request by cristianonicolai**
https://github.com/ansible/ansible-dev-tools/issues/35

### CODE REVIEW

**Review by shatakshiiii — APPROVED** [review]

File: docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Ansible Development Tools (ADT)
+
+## Introduction
+
+Ansible Development Tools or ADT for short, aims to streamline the setup and usage of several tools needed in order to create [Ansible] content.
+When it comes to create automation content using Ansible, there are several packages available that can help users in different parts of the content creating journey. From bootstrapping new projects, all the way to ensuring content follows best practices and verifying it behaves as intended via well established test frameworks.
```
Comment (ajinkyau) [inline docs/introduction.md:6]: When it comes to `creating` automation content

File: docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Key Features
+
+- All-in-One Ansible Toolkit: ADT combines critical Ansible development packages into a unified Python package.
+
+- Simplified Ansible Automation: With ADT, focus on crafting your automation scenarios instead of dealing with the intricacies of managing and integrating different Ansible libraries.
+
+- Boosted Development Efficiency: By bundling essential Ansible packages, ADT is designed to accelerate development speed and reduce boilerplate code. Spend less time setting up and more time creating powerful automation workflows.
```
Comment (ajinkyau) [inline docs/introduction.md:14]: can we combine points (2) `Simplified Ansible Automation` and (3) `Boosted Development Efficiency` as I think it is trying to convey more or less the same thing.

```
ADT focuses on crafting your automation scenarios and
workflows with speed by reducing boilerplate code without
dealing with the intricacies of managing and integrating
different Ansible libraries
```

**Review by ajinkyau — CHANGES_REQUESTED** [review]

File: docs/introduction.md (hunk 2 above)
Comment (cristianonicolai) [reply docs/introduction.md]: @ajinkyau sure, done

**Review by ajinkyau — APPROVED** [review]

File: docs/introduction.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  dealing with the intricacies of managing and integrating different Ansible libraries.
+
+For those looking for an IDE based experience, we also recommend you get familiar with the [Ansible extension for VSCode](https://marketplace.visualstudio.com/items?itemName=redhat.ansible).
+
+## Included Packages
+
+The curated list of tools installed as part of the Ansible Development Tools includes:
+
+- [ansible-core](https://github.com/ansible/ansible): Ansible is a radically simple IT automation platform that makes your applications and systems easier to deploy and maintain. Automate everything from code deployment to network configuration to cloud management, in a language that approaches plain English, using SSH, with no agents to install on remote systems.
+- [ansible-builder](https://github.com/ansible/ansible-builder): Ansible Builder is a tool that automates the process of building execution environments using the schemas and tooling defined in various Ansible Collections and by the user.
```
Comment (ariordan-redhat) [inline docs/introduction.md:22]:
```suggestion
- [ansible-builder](https://github.com/ansible/ansible-builder): Ansible Builder automates the process of building execution environments using the schemas and tooling defined in various Ansible Collections and by the user.
```

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**